README.md

```diff
@@ -221,16 +221,86 @@ root
 </pre>
 
 
-
-## Usage
-Todo
-
-### Compiling
-
-- Install go version > 1.8
-
-### Compiling
-
+## Getting Started
+The following section describes building the reference implementation of the Obscuro protocol, running the unit and 
+integration tests, and deploying a local testnet for end-to-end testing. The reference implementation of Obscuro is 
+written in [go](https://go.dev) with the IDE of choice being [GoLand](https://www.jetbrains.com/go/) from Jetbrains. 
+Unless otherwise stated, all paths stated herein are relative to the root of the `go-obscuro` checkout.
+
+
+### Dependencies
+The following dependencies are required to be installed locally;
+
+- [go](https://go.dev) (version > 1.8)
+- [docker](https://docs.docker.com/get-docker/) (recommend latest version)
+
+
+### Building
+To create the build artifacts local to the checkout of the repository the easiest approach is to build each component 
+separately for the host, enclave, and wallet extension  i.e. 
+
+```
+cd ./go/host/main && go build && cd -
+cd ./go/enclave/main && go build && cd -
+cd ./tools/walletextension/main && go build && cd -
+```
+
+Running `go build ./...` to build all packages at the root level will build all packages, but it will discard the 
+resulting artifacts; it therefore serves only as a check that the packages _can_ be built. Note that building the 
+enclave using `go` will compile it for a non-SGX mode and allow it to be run for test purposes. Compiling for SGX mode 
+requires `ego-go ` from [Ego](https://www.edgeless.systems/products/ego/) to be used in placement.  
```

Comment:
maybe worth mentioning here that we use docker and run ego-go inside it

Reply:
Done.